src/main/java/Duke.java

```diff
@@ -1,10 +1,101 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
+    /**
+     * Prints out all the items in the inventory
+     *
+     * @param inventory inventory of items added
+     */
+    public static void listInventory(ArrayList<Task> inventory){
+        for(int i = 0; i < inventory.size(); i++){
+            System.out.print(i+1 +".");
+            System.out.println(inventory.get(i));
+        }
+        System.out.println();
+    }
+
+    /**
+     * Mark item in the inventory as checked
+     *
+     * @param index index of item in inventory
+     * @param inventory inventory of all items
+     */
+    public static void checkInventory(int index, ArrayList<Task> inventory){
+        inventory.get(index).setIsChecked(true);
+        System.out.println("Nice! I've marked this task as done: ");
+        System.out.println(inventory.get(index)+"\n");
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String intro = "Hello! I'm Duke\n" +
+                "What can I do for you?\n";
+        String bye = "Bye. Hope to see you again soon!\n";
+        ArrayList<Task> inventory = new ArrayList<Task>(100);
+
+        System.out.println(intro);
+        Scanner in = new Scanner(System.in);
+        String line = in.nextLine();
+        String by;
+        String at;
+
+        // Takes in user input until "bye" is entered by user
+        while (!line.equals("bye")){
+            // List inventory
+            if (line.equals("list")){
+                listInventory(inventory);
+            }else if(line.startsWith("done ")){
+                // check items
+                line = line.replace("done ","");
+                try{
+                    int index = Integer.parseInt(line) -1;
+                    checkInventory(index, inventory);
+                }catch(Exception e){
```

Comment:
I like that you have handled exception, however, should we be more specific with the error message? So that the user can be more well informed that he/she did not input an integer. 

Suppose:
instead of printing "Opps Error", maybe can print "marked index not found/invalid"